service/frontend/nexus_handler.go

```diff
@@ -167,9 +169,14 @@ func (h *nexusHandler) getOperationContext(ctx context.Context, method string) (
 	}
 	oc := operationContext{nexusContext: nc, auth: h.auth, cleanupFunctions: make([]func(), 0)}
 
-	oc.metricsHandlerForInterceptors = h.metricsHandler.WithTags(metrics.OperationTag(nc.apiName), metrics.NamespaceTag(nc.namespaceName))
+	oc.metricsHandlerForInterceptors = h.metricsHandler.WithTags(
+		metrics.OperationTag(nc.apiName),
+		metrics.NamespaceTag(nc.namespaceName),
+		metrics.NexusServiceTag(nc.serviceName),
```

Comment:
You can't use this in the interceptors unfortunately. The tags need to be consistent with gRPC.